frontend/src/js/components/subscription/SubscriptionPage.tsx

```diff
@@ -57,6 +57,14 @@ const contactReasons = {
         If you want to reduce your device limit, please contact <SupportLink variant="email" />.
       </div>
     )
+  },
+  specialHandling: {
+    id: 'specialHandling',
+    alert: (
+      <div>
+        You have a special agreement with Northern.tech, please contact <SupportLink variant="email" /> for any change in your subscription.
+      </div>
+    )
```

Comment:
The non-trial customers that does not have a Stripe subscription should fall back into the same handling as enterprise customers. That is, they will get the form to request a quote. This should bind to the current API to request plan change `POST /api/management/v2/tenantadm/tenants/{id}/plan`.

Reply:
The enterprise form you described is already shown. If it won't be confusing for a user which effectively have Professional or Basic plan, but shown enterprise form, I could remove this Alert information or change its description. What would you suggest?

Reply:
The input should select the enterprise form and disable the radio group as soon as GET `/billing/subscription` fails with 404, I'll double-check that on my side, but could you please confirm that this requests fails as expected in the tenant service when you test it?